sql/parser/sql.y

```diff
@@ -1402,11 +1402,11 @@ rename_stmt:
   }
 | ALTER INDEX qualified_name RENAME TO name
   {
-    $$ = nil
+    $$ = &RenameIndex{Name: $3, NewName: Name($6), IfExists: false}
   }
```

Comment:
in postgres, `ALTER INDEX` is just an alias for `ALTER TABLE ...` with indices. Should we implement the latter as well?